spec/rails_helper.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
```

Comment:
- [ ] For the RSpec tests, I see you might have missed that. So, please add unit and integration tests for your app's funcionalities with covering both negative and positive scenarios. 🙏

Reply:
<img width="419" alt="Captura de Pantalla 2022-08-06 a la(s) 17 22 58" src="https://user-images.githubusercontent.com/13943949/183257547-2d9dba89-fb10-44e7-9ed5-b3b6d8c8d418.png">

Tests added and running 👍